### PR TITLE
TST/REF: Collect Reduction, Arithmetic Tests

### PR DESCRIPTION
**Pull request by jbrockmendel**
Remove unused attributes in test_nanops

### CODE REVIEW

File: pandas/tests/test_nanops.py [hunk 1]
```diff
@@ -77,51 +77,19 @@ def setup_method(self, method):
 
         self.arr_float_2d = self.arr_float[:, :, 0]
         self.arr_float1_2d = self.arr_float1[:, :, 0]
-        self.arr_complex_2d = self.arr_complex[:, :, 0]
```
Comment (jreback) [inline pandas/tests/test_nanops.py]: these are de-duplicating?

**Review by jreback — CHANGES_REQUESTED** [review]
these are all just moves?

**Comment by jbrockmendel** [comment]
> these are all just moves?

In some places I think I changed pytest.raises to use the context syntax.  No material changes.

File: pandas/tests/test_nanops.py (hunk 1 above)
Comment (jbrockmendel) [reply pandas/tests/test_nanops.py]: just never used

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks!

**Pull request merged**